src/views/course_pairings.vue

```diff
@@ -0,0 +1,239 @@
+<style>
+@import "../../node_modules/handsontable/dist/handsontable.full.css";
+#table-format {
+    width: 90%;
+    height: 600px;
+    margin: auto;
+    overflow: hidden;
+    border: 2px solid #09b4ed;
+}
+</style>
+
+<template>
+
+<div>
+    <h1>Course and Section Data</h1>
+    <div>
+        <button>Import Data</button>
+        <button>Download Sample Data</button>
+
+        <p>Specify Course: </p>
+        <select v-model="specify_course">
+            <option>All Courses</option>
+            <option v-for="course in this.list_courses">{{course}}</option>
```

Comment:
`list_courses` sounds like an action, so I would expect `this.list_courses` to be a function. Call it `this.courses` or `this.courses_list` if it is just a list.